docs/references/kontena-yml.md

```diff
@@ -471,33 +473,36 @@ db:
 ```
 
 ## Example kontena.yml
+
 ```
-loadbalancer:
-  image: kontena/lb:latest
-  ports:
-    - 80:80
-app:
-  build: .  
-  image: registry.kontena.local/example-app:latest
-  instances: 2
-  privileged: true  
-  links:
-    - loadbalancer
-  environment:
-    - DB_URL=%{project}-db.kontena.local
-    - KONTENA_LB_INTERNAL_PORT=80
-    - KONTENA_LB_VIRTUAL_HOSTS=www.my-app.com
-  deploy:
-    strategy: ha
-    wait_for_port: 80
-  hooks:
-    post_start:
-      - name: sleep
-        cmd: sleep 10
-        instances: *  
-db:
-  image: mysql:5.6
-  stateful: true
-  volumes:
-    - /var/lib/mysql
+version: "2"
+services:
+  loadbalancer:
+    image: kontena/lb:latest
+    ports:
+      - 80:80
+  app:
+    build: .  
+    image: registry.kontena.local/example-app:latest
+    instances: 2
+    privileged: true  
```

Comment:
While editing this example, could we just remove this `privileged` option. It's not the most common option.